packages/core/scripts/local/RequestOraclePrice.js

```diff
@@ -61,13 +62,24 @@ async function run(deployedFinder, identifier, timeString) {
       await deployedFinder.getImplementationAddress(web3.utils.utf8ToHex(interfaceName.IdentifierWhitelist))
     );
     await deployedIdentifierWhitelist.addSupportedIdentifier(identifierInBytes);
-    const priceExists = await deployedVoting.hasPrice(identifierInBytes, timeInBN);
+    let priceExists;
+
+    if (ancillaryDataInBytes) {
+      priceExists = await deployedVoting.hasPrice(identifierInBytes, timeInBN, ancillaryDataInBytes);
+    } else {
+      priceExists = await deployedVoting.hasPrice(identifierInBytes, timeInBN);
```

Comment:
this script asks for new prices. im allowing this script to run either function to ensure when testing voter dapp both situations work ( and it does)

Reply:
depends on what you mean by does it work. I've updated the test script to create price requests that hit both of these branches, and it works as expected: price requests are made with and without ancillary data. Its possible that line 70 is still calling the ancillary version of the function though, which could be giving me similar behavior.